src/polytopes/simplex.jl

```diff
@@ -0,0 +1,69 @@
+"""
+    Simplex(p1, p2, ..., pK, pK+1)
+
+A K-simplex is a "simplical" polytope that occupies space in K dimensions using K+1 points,
+and is embedded in some dimension `Dim`.
+For example, a line segment in 3-D space would be Simplex{1, 3} as it occupies a single dimension,
+is made up of two points, and lies in 3d space.
+
+Note that we only allow constructors of the form `Simplex(p1,p2,...,pk+1)`, not `Simplex(Tuple(...))`.
```

Comment:
```suggestion
```

Reply:
We need to follow the same convention of Triangle, Tetrahedron, etc. If they can be constructed with Tuple, then Simplex should be too.